include/repgp/repgp_def.h

```diff
@@ -94,8 +95,17 @@
 #define PGP_KEY_ID_SIZE 8
 
 /* Size of the fingerprint */
-#define PGP_FINGERPRINT_SIZE 20
-#define PGP_FINGERPRINT_HEX_SIZE (PGP_FINGERPRINT_SIZE * 2) + 1
+#define PGP_FINGERPRINT_V4_SIZE 20
```

Comment:
Please do not rename this to have less code changes and leave as `PGP_FINGERPRINT_SIZE`.

Reply:
It appears that such single comments do not have 'resolve' button. Will post further comments as part of the review in a batch.

Reply:
We can readily change that as suggested, but just as another possible aspect to consider: wouldn't it in the long run be rather worth to have some more code changes now and later have a clearer naming scheme for these defines? What you request to be kept named `PGP_FINGERPRINT_SIZE` might later be an outdated version thus lead to misinterpretation of the source code.